ehitchhiking-ui/src/app/app.component.sass

```diff
@@ -1 +1,14 @@
+html,
+body,#map
+  width: 100%
+  height: 100%
+  padding: 0
+  margin: 0
+  overflow: hidden
+  position: absolute
 
+.registration
+  width: 100%
+  position: relative
+  z-index: 5
```

Comment:
Could you please clarify why z-index: 5 is required here ?

Reply:
I remove .registation . It was not necessary